java/src/main/c++/jni_spark_vw.cc

```diff
@@ -123,12 +202,14 @@ JNIEXPORT jbyteArray JNICALL Java_org_vowpalwabbit_spark_VowpalWabbitNative_getM
   catch (...)
   {
     rethrow_cpp_exception_as_java_exception(env);
+    // return null
+    return 0;

```

Comment:
nullptr?

Reply:
Will do.

Went and checked and jbytearray is defined as:
```C++
typedef _jbyteArray *jbyteArray;
```

Reply:
theres some others, but not all of them. i saw a long in there or something but it also might be a pointer